### PR TITLE
moving to go mod

### PR DESCRIPTION
**Pull request by surajnarwade**
With this PR, we are updating golang version in dronefile and moving to go modules

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -4,7 +4,9 @@ workspace:
 
 pipeline:
   build:
-    image: golang:1.10
+    image: golang:1.14
```
Comment (waiariki-koia) [inline .drone.yml:7]: Does this need to match the version in go.mod?

File: .drone.yml [hunk 2]
```diff
@@ -4,7 +4,9 @@ workspace:
...
+    environment:
+      - GO111MODULE=on
```
Comment (waiariki-koia) [inline .drone.yml:9]: If using go 1.16, this is set by default

File: .drone.yml (hunk 1 above)
Comment (surajnarwade) [reply .drone.yml:7]: oh sorry, that's my local Golang. will change it, thought of keeping it stable version of golang and not latest.

File: .drone.yml (hunk 2 above)
Comment (surajnarwade) [reply .drone.yml:9]: I don't intend to use 1.16 for now

**Review by waiariki-koia — APPROVED** [review]

**Pull request merged**